packages/manager/cypress/e2e/core/notificationsAndEvents/qemu-reboot-upgrade-notice.spec.ts

```diff
@@ -0,0 +1,343 @@
+import {
+  capitalize,
+  linodeConfigInterfaceFactory,
+  linodeFactory,
+  profileFactory,
+} from '@linode/utilities';
+import { notificationFactory } from '@src/factories/notification';
+import { mockGetAccount, mockGetMaintenance } from 'support/intercepts/account';
+import { mockGetLinodeConfigs } from 'support/intercepts/configs';
+import { mockGetNotifications } from 'support/intercepts/events';
+import {
+  mockGetLinodeDetails,
+  mockGetLinodes,
+  mockGetLinodeVolumes,
+} from 'support/intercepts/linodes';
+import { mockGetProfile } from 'support/intercepts/profile';
+import { mockGetVLANs } from 'support/intercepts/vlans';
+import { randomIp, randomLabel, randomNumber } from 'support/util/random';
+import { chooseRegion } from 'support/util/regions';
+
+import {
+  accountFactory,
+  accountMaintenanceFactory,
+  linodeConfigFactory,
+  VLANFactory,
+  volumeFactory,
+} from 'src/factories';
+
+import type { Linode, Notification } from '@linode/api-v4';
+
+describe('QEMU reboot upgrade notification', () => {
+  const noticeMessageShort =
+    'One or more Linodes need to be rebooted for critical platform maintenance.';
+  const noticeMessage = `${noticeMessageShort} See which Linodes are scheduled for reboot on the Account Maintenance page.`;
+  const rebootReason =
+    'This maintenance is scheduled to upgrade the QEMU version.';
+
+  const notifications: Notification[] = [
+    notificationFactory.build({
+      severity: 'major',
+      type: 'security_reboot_maintenance_scheduled',
+      message: noticeMessageShort,
+      label: 'QEMU Reboot Upgrade Notice',
+    }),
+  ];
+
+  /**
+   * This test verifies that the QEMU reboot upgrade notice is displayed in the Linode landing page.
+   *
+   * - Check that the notice is visible and contains the expected message.
+   * - Check that user gets notified and the notification is present in the notifications dropdown.
+   * - Check that a maintenance help button is displayed near the status of impacted Linodes.
+   */
+  it(`should display maintenance banner in 'Linnode' landing page when one or more Linodes get impacted.`, () => {
+    const mockAccount = accountFactory.build();
+    const mockProfile = profileFactory.build({
+      restricted: false,
+      username: 'mock-user',
+    });
+    const mockLinodes = new Array(5)
+      .fill(null)
+      .map((item: null, index: number): Linode => {
+        return linodeFactory.build({
+          label: `Linode ${index}`,
+          region: chooseRegion({
+            capabilities: ['Linodes', 'Vlans'],
+          }).id,
+        });
+      });
```

Comment:
Optional: We may be able to use `linodeFactory.buildList` here

```suggestion
    const mockLinodes = linodeFactory.buildList(5, {
      region: chooseRegion({
        capabilities: ['Linodes', 'Vlans'],
      }).id,
    });
```